lib/funky/version.rb

```diff
@@ -1,3 +1,3 @@
 module Funky
-  VERSION = "0.2.9"
+  VERSION = "0.3.0"
```

Comment:
@bruce-fs Backward-compatible changes only bump the patch version when version < 1.

So this should be `"0.2.10"`